FAQ.md

```diff
@@ -841,3 +973,7 @@ This error usually occurs when the version of TiDB does not match with the versi
 When you execute the `LOAD DATA LOCAL` statement but the MySQL client does not allow executing this statement (the value of the `local_infile` option is 0), this error occurs.
 
 The solution is to use the `--local-infile=1` option when you start the MySQL client. For example, use command like `mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000`. The default value of `local-infile` is different in different versions of MySQL client, therefore you need to configure it in some MySQL clients and do not need to configure it in some others.
+
+#### ERROR 9001 (HY000): PD server timeoutstart timestamp may fall behind safepoint
+
+This error occurs when TiDB accesses PD. A worker in the TiDB background continuously queries the safepoint in PD and this error occurs if it fails to query within 100s. Generally it is because PD failure or the network problem between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](sql/error.md).
```

Comment:
- when TiDB fails to access PD (not always occur; we should make it more logical)
- A worker in the TiDB background continuously queries the safepoint in PD -> A worker from PD continuously queries the safepoint in the TiDB background
- PD failure or the network problem -> the PD failure or network failure